runtime/README.md

```diff
@@ -0,0 +1,488 @@
+| 0x1C        | 31    | Internal TCI   | This TCI was measured by Runtime Firmware itself
+|             | 30:0  | Reserved       | Reserved flag bits
+| 0x20        | 383:0 | Latest TCI     | The latest `INPUT_DATA` extended into this TCI by ExtendTci or DeriveChild
+
+### Certificate Generation
+
+DPE leaf certs are mostly fixed size, but do have some variable-size sections:
+
+* tcg-dice-MultiTcbInfo extension
+* Signature
+
+For this reason, DPE must do some ASN.1 generation. An implementation MAY
+choose to template certain structures in the certificate for convenience.
+
+The DPE Runtime Alias Key SHALL sign DPE leaf certificates.
+
+The DPE `GET_CERTIFICATE_CHAIN` command shall return the following certificates:
+
+* IDevID
+* LDevID
+* FMC Alias
+* Runtime Alias
+
+### DPE Leaf Certificate Definition
+
+| Field                          | Sub Field   | Value
+| -------------                  | ---------   | ---------
+| Version                        | v3          | 2
+| Serial Number                  |             | First 20 bytes of sha256 hash of DPE Alias public key
+| Issuer Name                    | CN          | Caliptra Runtime Alias
+|                                | serialNumber | First 20 bytes of sha384 hash of Runtime Alias public key
+| Validity                       | notBefore   | February 10th, 2023
+|                                | notAfter    | 99991231235959Z
+| Subject Name                   | CN          | Caliptra DPE Leaf
+|                                | serialNumber | SHA384 hash of Subject public key
+| Subject Public Key Info        | Algorithm   | ecdsa-with-SHA384
+|                                | Parameters  | Named Curve = prime384v1
+|                                | Public Key  | DPE Alias Public Key value
+| Signature Algorithm Identifier | Algorithm   | ecdsa-with-SHA384
+|                                | Parameters  | Named Curve = prime384v1
+| Signature Value                |             | Digital signature for the certificate
+| KeyUsage                       | keyCertSign | 1
+| Basic Constraints              | CA          | False
+| Policy OIDs                    |             | id-tcg-kp-attestLoc
+| tcg-dice-MultiTcbInfo\*        | Vendor      | {Vendor-defined}
+|                                | Model       | Caliptra
+|                                | SVN         | 1
+|                                | FWIDs       | [0] "Journey" TCI Value
+|                                |             | [1] "Current" TCI Value. Latest `INPUT_DATA` made by DeriveChild or ExtendTci.
+|                                | Type        | 4-byte TYPE field of TCI node
+|                                | VendorInfo  | Locality of the caller (analog for PAUSER)
+
+\*MultiTcbInfo ontains one TcbInfo for each TCI Node in the path from the
+current TCI Node to the root. Max of 24.
+
+# Opens
+
+The following items are still under discussion
+
```

Comment:
Summarizing my requested opens since I pressed the wrong github button:
* Firmware authentication vs just ECDSA verify as the API
* Handling of impactless updates when firmware + manifest size exceeds mailbox size
* Resource usage https://github.com/chipsalliance/caliptra-sw/issues/17

Reply:
Covered these in opens